js/history.js

```diff
@@ -41,6 +104,9 @@ Fliplet.Registry.set('history-stack', (function() {
     if (stack.length > 1) {
       currentIndex++;
     }
+    
+    // store columns info
```

Comment:
Start comments with correct capitalisation

```suggestion
    // Store columns info
```